index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <div class="app-bar">
+        <div class="upper-bar">
+          <p class="time">9 : 41</p> 
+          <div class="upper-icons">
+            <p class="ntew"> </p>
+            <p class="wifi"></p>
+            <p class="battery"></p>
+          </div>
+        </div>
+        <div class="lower-bar">
+          <p class="logoo"><img src="image/Logo.png" alt="logo icon"></p> 
```

Comment:
- [ ] Kindly wrap your logo in an `a` tag and change it to a `text`, not an image 🙏 . This would make your code semantic and give a better user experience(UX)